README.md

```diff
@@ -1,2 +1,18 @@
-# dfaker
-A generator of fake diabetes data.
+#Tidepool Dfaker 
+
+dfaker is a fake data generator that simulates blood glucose data over time. 
+**Note** This project is under active development 
+
+#Set up
+
+dfaker runs with python3. In addition, Python Anaconda is required. 
+The anaconda installation for python3 can be [found here](http://continuum.io/downloads#py34).
+
+#Credits
+
+The implementation for the blood sugar model equation was based on a mathematics paper on [blood glucose levels](http://scholarcommons.usf.edu/cgi/viewcontent.cgi?article=4830&context=ujmm). 
```

Comment:
Consistency - change to blood glucose since that's the term you use elsewhere ;)